### PR TITLE
chore (events) November 2021, part 2

### PR DESCRIPTION
**Pull request by TinaNEs**
added new events

### CODE REVIEW

File: data/events.xml [hunk 1]
```diff
@@ -1,5 +1,141 @@
...
+        <speaker>Brandy Chang</speaker>
+        <title>SJCConf 2021</title>
+        <subject>Going Reactive with Kotlin and Quarkus</subject>
+        <url>https://jcconf.tw/2021/</url>
+        <description>
+            <![CDATA[
+<p>Introduce Quarkus for cloud native development.  Demo to build small and high concurrent restful service with Kotlin coroutines and Quarkus.<p>
+]]>
+        </description>
+      </event>
```
Comment (meilalina) [inline data/events.xml:68]: @TinaNEs not sure if you saw it, but there is at least one more talk about KMM. Could you please ad it too?

File: data/events.xml [hunk 2]
```diff
@@ -1,5 +1,141 @@
...
+<p>This live coding kata has 2 goals:
+1) experience how TDD flow feels like (although we cheat a bit, as the unit tests are already written for us)
+2) play with Kotlin, and be amazed by how functional programming feels in that language.<p>
+]]>
+        </description>
+     </event>
+
+     <event>
+        <lang>en</lang>
+        <startDate>2021-02-09</startDate>
```
Comment (meilalina) [inline data/events.xml]: @TinaNEs wrong date

File: data/events.xml [hunk 3]
```diff
@@ -1,5 +1,56 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <events>
+    <event>
+        <lang>zh</lang>
```
Comment (meilalina) [inline data/events.xml:4]: @TinaNEs I think the language mark is wrong

File: data/events.xml [hunk 4]
```diff
@@ -1,5 +1,56 @@
...
+        <url>https://www.meetup.com/bld-tech-talks/events/281224568</url>
+        <description>
+            <![CDATA[
+<p>In this session, we look at how companies can share business logic between platforms while only writing code once and then running it on all platforms. We look at how such an architecture reduces risk, increases productivity and how this can be achieved using a single programming language. We look at hybrid approaches that might be more pragmatic in existing teams and the benefits of going this route. We also look at why Kotlin is such a compelling choice for achieving architectural nirvana.<p>
+]]>
+        </description>
+     </event>
+
+     <event>
+        <lang>zh</lang>
```
Comment (meilalina) [inline data/events.xml:20]: @TinaNEs the language mark looks wrong here too

File: data/events.xml [hunk 5]
```diff
@@ -1,5 +1,56 @@
...
+        <description>
+            <![CDATA[
+<p>Kotlin is a modern language targeting the JVM ecosystem and is becoming more and more attractive for Java developers.
+In this talk we will give you a first insight into Kotlin. We are also going to take a look why you as a Java developer should care for this language and what every Java developer should know about Kotlin.<p>
+]]>
+        </description>
+     </event>
+
+     <event>
+        <lang>zh</lang>
```
Comment (meilalina) [inline data/events.xml:37]: @TinaNEs language mark

File: data/events.xml (hunk 1 above)
Comment (TinaNEs) [reply data/events.xml:68]: added 2 more talks with Kotlin

**Pull request merged**